index.html

```diff
@@ -0,0 +1,147 @@
+          <nav class="navigation">
+            <div class="menu-button"><img id="humberger" src="img/menu.svg" alt="menu-button"></div>
+            <a href="index.html"><img class="logo"src="img/logo-2.png" alt="Manage" /></a>
+          <div class="nav-bar">
+          <ul  class="nav-list text-neutral-900 fw-bold fs-500" >
+          <li><a class="nav-link"href="index.html">Home</a></li>
+          <li><a class="nav-link" href="about-page.html">About</a></li>
+          <li><a class="nav-link" href="#courses">Courses</a></li>
+          <li><a class="nav-link" href="#instructor">Instrictors</a></li>
+        </ul>
+          </div>
+        
+      </nav>
+    </div>
+  </header>
+  <div id="nav-wrap">
+    <div id="close-button"><img src="img/x-icon.svg" alt="close-button"></div>
+    <div id="nav-m-wrap">
+      <li><a href="index.html">Home</a></li>
+      <li><a href="about-page.html">About</a></li>
+      <li><a href="#courses">Courses</a></li>
+      <li><a href="#instructor">Instrictors</a></li>
+    </div>
+  </div>
+  <main>
+    <section id="hero">
+      <div class="container">
+        <div>
+          
+            <h2 class="fs-secondary-heading fw-bold text-accent-400 ">"Hello! future Devs"</h2>
+            <div class="hero-headline">  
+            <h1 class="hero-headline text-accent-400 ">Break into tech with <br>the tech academy </h1>
+          </div>
+          
+          <div class="offer-text">
+            <p class="fs-600">We offer open enrollment Courses, which means students can begin <br>studying at any time, from anywhere.</p>
+          </div>
+            <h2 class="fs-secondary-heading text-neutral-100 fw-bold ">September- December 2022</h2>
+            <h3 class="fs-600 text-neutral-900">20% off the course fee from September- December 2022!</h2>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <section id="courses">
+      <div class="container">
+        <div>
+          <h2 class="title fs-700 text-accent-100 fw-semi-bold" >Our Courses</h2>
+          <div class="guide_bar"></div>
+        </div>
+        <div class="even-columns-course">
+          <div class="course-card">
+            <img class="course-icon" src="img/html-5.png" alt="Html-icon">
+            <h3 class="fw-bold fs-500">HTML</h3>
+            <p class="course-desc">Lorem ipsum dolor, sit amet consectetur adipisicing elit.</p>
+          </div>
+          <div class="course-card">
+            <img class="course-icon"src="img/css-3.png" alt="Css-icon">
+            <h3 class="fw-bold fs-500">CSS</h3>
+            <p class="course-desc">Lorem ipsum dolor, sit amet consectetur adipisicing elit.</p>
+          </div>
+          <div class="course-card">
+            <img class="course-icon" src="img/bootstrap.png" alt="bootstrap-icon">
+            <h3 class="fw-bold fs-500">Bootstrap</h3>
+            <p class="course-desc">Lorem ipsum dolor, sit amet consectetur adipisicing elit.</p>
+          </div>
```

Comment:
### Your Cards
![Screenshot from 2022-10-20 15-19-16](https://user-images.githubusercontent.com/93039359/196923338-77c30f49-ad54-4f75-8ce2-e9af24419fda.png)
### Design Cards
![image](https://user-images.githubusercontent.com/93039359/196924081-4c500f99-35b8-45ee-9ea4-130121e34888.png)

- [ ] You can compare both images and see that in the mobile version then contents should be in horizontal pattern but in your design contents are in vertical pattern. Kindly make you contents in horizotal means left to right not top to bottom :+1: